torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3947,7 +3947,10 @@ def sample_inputs_nextafter(op_info, device, dtype, requires_grad, **kwargs):
     cases = (
         ((S, S), (S, S), False),
         ((S, S), (S,), False),
-        ((S, ), (S, S), True)
+        ((S, ), (S, S), True),
```

Comment:
You should undo the changes here.

Sorry I forgot to mention, you will have to update the following test (since NumPy doesn't have nextafter which supports bfloat16, we had to hand-roll a test with golden values)

https://github.com/pytorch/pytorch/blob/913c1f83f49f9e1e2a494186cc0069d780cee852/test/test_binary_ufuncs.py#L1802-L1804

Reply:
Done, thanks.